cli/args/mod.rs

```diff
@@ -517,6 +512,28 @@ impl CliOptions {
     self.maybe_config_file.as_ref().map(|f| f.specifier.clone())
   }
 
+  // TODO(Cre3per): In deno_ast
+  // - Change MediaType::map_js_like_extension() to operate on extension, not path
+  // - Add MediaType::to_content_type() (This function, but complete)
+  fn media_type_from_file_extension(extension: &str) -> Option<String> {
+    match extension {
+      "ts" => Some("application/typescript"),
+      "tsx" => Some("text/tsx"),
+      "js" => Some("application/javascript"),
+      "jsx" => Some("text/jsx"),
+      _ => None,
```

Comment:
Perhaps we should error if the extension is not understood? This will allow people who accidentally provide an incorrect extension to not wonder why it's not working and get direct feedback.

Reply:
By "people", do you mean developers or users?  
Developers get feedback via `Option`. Users cannot provide invalid extensions because clap restricts the allowed values of `--ext`.

If we want to support unfiltered user input for `--ext` in the future, we cannot use media types for `--ext` handling, because some extensions share media types but need different handling, eg. ".ts" and ".d.ts". Details in https://github.com/denoland/deno/pull/17172#issuecomment-1431643714

Related: This function moves to `deno_ast` with https://github.com/denoland/deno_ast/issues/133

Reply:
Ah, ok I always forget that clap is used to restrict inputs in this file. That's what I meant. Thanks!